nixos/modules/programs/openvpn3.nix

```
@@ -0,0 +1,33 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.openvpn3;
+in
+{
+  options.programs.openvpn3 = {
+    enable = mkEnableOption "openvpn3 client";
+  };
+
+  config = mkIf cfg.enable {
+    services.dbus.packages = [
+      pkgs.openvpn3
+    ];
```

Might as well be consistent.
```suggestion
    services.dbus.packages = with pkgs; [
      openvpn3
    ];
```